assets/src/edit-story/migration/migrations/v0006_setOpacity.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function setOpacity({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement({ opacity, ...rest }) {
+  return {
+    opacity: opacity ? opacity : 100,
```

Comment:
What if `opacity` is `0`?

Reply:
Hmm, currently the min and max value for the control are defined from 1-100 in the atts -- should it ever be allowed to be 0?

Reply:
Ah, didn't realize that. All good then 🙂 

Tests will reflect that too I suppose 👍

Reply:
Added some extra tests in 42b6e55.